### DISCUSSION TITLE
possible mobile port 👉👈

### DISCUSSION THREAD

**Discussion by Ra1nnYT** [nzp-team/nzportable#385] [General]
Maybe? IDK I don't wanna emulate it.

**Comment by Gokur16** [comment]
Would love to have this fsfs def good to emulate on mobile and add a controller

**Comment by TheMargwa** [comment]
would love a mobile port, .ipa or .apk in releases with touch controls or controller support would be amazing esp since this is nz portable

**Comment by MotoLegacy** [comment]
Hi, just thought I'd chime in here. I get this a lot and the TL;DR is the engine used as a base for the Desktop versions offers a native Android binary, but it is mostly untested. Additionally — I'm not really interested in supporting/accommodating mobile platforms anyway. Not only is it a lot of additional work to make a touch-friendly UI, I already maintain 13 other platforms. I'm not sure why you are not willing to emulate it, especially with YuZu available on Android now, but regardless I apologize if you do not feel accommodated.

**Reply by TheMargwa** [reply]
Thank you was just curious. I’m plenty accommodated with all the options thank you for all your hard work in the zombies community.
